hazelcast/src/main/java/com/hazelcast/nio/serialization/compact/CompactReader.java

```diff
@@ -37,7 +37,7 @@
  * The way to use {@link CompactReader} for class evolution is to check for the
  * existence of a field with its name and kind, with the
  * {@link #getFieldKind(String)} method. One should read the field, if it exists
```

Comment:
Can you remove this comma as well? I think it is not correct and makes understanding harder.